app/2.0/start/toolbox/deploy.md

```diff
@@ -143,10 +162,33 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
     ]
     ```
 
+    For example, your `firebase.json` file may look like this afterwards:
+	
+    ```
+    {
+      "database": {
+        "rules": "database.rules.json"
+      },
+      "hosting": {
+        "public": "build/default",
+        "rewrites": [
+          {
+            "source": "!/__/**",
+            "destination": "/index.html"
+          },
+          {
+            "source": "**/!(*.js|*.html|*.css|*.json|*.svg|*.png|*.jpg|*.jpeg)",
```

Comment:
is there a more general `*.*` that we could suggest to users here to check for all URLs that don't include file extensions?